File: hoja7/rectangulo.php
<?php
//error_reporting(E_ALL);
class Rectangulo {
    public $base;
    public $altura;
    public function calcularArea() {
        echo 'La Área cuyo base de '.$this->base. ' y altura '.$this->altura. ' es de '.$this->base * $this->altura;
    }
}

$area = new Rectangulo;
$area -> base = 10;
$area -> altura = 5;
$area -> calcularArea();
?>

File: Java4/Ejercicio_39b/calcularFiguras/Rectangulo.java
package calcularFiguras;

public class Rectangulo extends Figura{
	double base, altura;
	
	public Rectangulo(double base, double altura) {
		this.base = base;
		this.altura = altura;
	}
	
	@Override
	double calcularArea() {
		return base * altura;
	}
}
